src/Linux/Avalonia.LinuxFramebuffer/Output/FbdevOutput.cs

```diff
@@ -16,16 +16,23 @@ public sealed unsafe class FbdevOutput : IFramebufferPlatformSurface, IDisposabl
         private IntPtr _mappedAddress;
         public double Scaling { get; set; }
 
-        public FbdevOutput(string fileName = null)
+        /// <summary>
+        /// Create a Linux frame buffer device output
+        /// </summary>
+        /// <param name="format">The required pixel format for the frame buffer.
+        /// A null value will leave the frame buffer in the current pixel format.
+        /// Otherwise sets the frame buffer to the required format</param>
+        /// <param name="fileName">the frame buffer device name</param>
+        public FbdevOutput(PixelFormat? format = null, string fileName = null)
```

Comment:
Keep the original constructor so it could be merged to 0.10.x. Also it's probably better to have configuration options after the file name.

Reply:
OK, will do

Reply:
Done